### PR TITLE
install/update docs review

### PR DESCRIPTION
**Pull request by kalexand-rh**

### CODE REVIEW

File: modules/installation-network-user-infra.adoc [hunk 1]
```diff
@@ -21,33 +21,45 @@ addresses and host names to the cluster machines.
...
-|Host level services, including the node exporter on ports `9100`-`9101` and
-the Cluster Version Operator on port `9099`.
-
 |`10249`-`10259`
 |The default ports that Kubernetes reserves
 
 |`10256`
 |openshift-sdn
 
+.3+|UDP
```
Comment (kalexand-rh) [inline modules/installation-network-user-infra.adoc]: @wking, do you know what these ports are for? I don't mind adding them, but they need descriptions.

File: modules/installation-network-user-infra.adoc [hunk 2]
```diff
@@ -21,33 +21,45 @@ addresses and host names to the cluster machines.
...
 |The default ports that Kubernetes reserves
 
 |`10256`
 |openshift-sdn
 
+.4+|UDP
+|`4789`
+|
+
+|`6081`
```
Comment (wking) [inline modules/installation-network-user-infra.adoc]: These are [VXLAN and GENEVE][1].

[1]: https://github.com/openshift/installer/blob/499f53fcbc89fdddd5541f00a9c97578bb7dbc6d/data/data/gcp/network/firewall.tf#L106-L109

File: modules/installation-aws-user-infra-requirements.adoc [hunk 3]
```diff
@@ -152,13 +154,15 @@ If `m4` instance types are not available in your region, such as with
...
+You might be able to use other instance types that meet the specifications of these instance types.
+
 [id="installation-aws-user-infra-other-infrastructure_{context}"]
 == Other infrastructure components
 
 * A VPC
 * DNS entries
 * Load balancers and listeners
-* A Route53 zone
+* A Public and a private Route53 zone
```
Comment (wking) [inline modules/installation-aws-user-infra-requirements.adoc]: nit: "Public" -> "public"

File: modules/installation-aws-user-infra-requirements.adoc [hunk 4]
```diff
@@ -193,12 +197,11 @@ and associate them with appropriate Ingress rules.
...
 * `AWS::EC2::RouteTable`
 * `AWS::EC2::Route`
-* `PublicSubnetRouteTableAssociation`
+* `AWS::EC2::SubnetRouteTableAssociation`
 * `AWS::EC2::NatGateway`
 * `AWS::EC2::EIP`
 2+|You must have a public internet gateway, with public routes, attached to the
-VPC. Each public subnet must also be attached to the route and have a NAT
-gateway and EIP address.
+VPC. In the provided templates, each public subnet is attached to the route and has a NAT gateway and EIP address. If you need to limit external access to your cluster, you can
```
Comment (wking) [inline modules/installation-aws-user-infra-requirements.adoc]: dangling sentence?

File: modules/installation-aws-user-infra-rhcos-ami.adoc [hunk 5]
```diff
@@ -18,48 +18,51 @@ You must use a valid {op-system-first} AMI for your Amazon Web Services
...
 |`ca-central-1`
-|`ami-0f907257d1686e3f7`
+|`ami-083a6c4d5fec9e7f7`
 
 |`eu-central-1`
-|`ami-02fdd627029c0055b`
+|`ami-00731d73fad047d6a`
+
+|`eu-north-1`
+|`ami-0359348d6d4ede973`
```
Comment (wking) [inline modules/installation-aws-user-infra-rhcos-ami.adoc]: RHCOS is here, but the installer doesn't support it yet: openshift/installer#2308.  Probably leave it off until we get that landed, since it's unlikely to get backported to 4.2, and leaving it off now will make it easier to backport docs to 4.2 (although the master AMIs have diverged from the 4.2 AMIs; looks like you're using the master AMIs here).

File: modules/installation-generate-aws-user-infra-ignition.adoc [hunk 6]
```diff
@@ -3,6 +3,14 @@
 // * installing/installing_aws_user_infra/installing-aws-user-infra.adoc
 // * installing/installing_restricted_networks/installing-restricted-networks-aws.adoc
 
+ifeval::["{context}" == "installing-restricted-networks-aws"]
+:restricted:
+:aws:
```
Comment (wking) [inline modules/installation-generate-aws-user-infra-ignition.adoc]: This is strange inside a `installation-generate-aws-user-infra-ignition.adoc` module.  Can we drop `:aws:` and `ifdef::aws[]` conditionals in this file?  Or rename to a generic `installation-generate-user-infra-ignition.adoc`?

File: modules/installation-launching-installer.adoc [hunk 7]
```diff
@@ -60,7 +60,7 @@ You can install {product-title} on a compatible cloud.
 
 [IMPORTANT]
 ====
-You can run the installation program only once, during initial installation.
+You can run the `create cluster` option of the installation program only once, during initial installation.
```
Comment (wking) [inline modules/installation-launching-installer.adoc]: nit: "option" -> "subcommand"?  Or just "command"?  Or "the installation program's `create cluster` only once"?  Or something ;).

File: modules/installation-network-user-infra.adoc [hunk 8]
```diff
@@ -21,33 +21,45 @@ addresses and host names to the cluster machines.
...
+.4+|TCP
 |`2379`-`2380`
 |etcd server, peer, and metrics ports
 
 |`6443`
 |Kubernetes API
 
-|`9000`-`9999`
-|Host level services, including the node exporter on ports `9100`-`9101` and
-the Cluster Version Operator on port `9099`.
```
Comment (wking) [inline modules/installation-network-user-infra.adoc]: These are [open on TCP too][1], so I think we want to keep this entry.  I dunno whether the node exporter and CVO stuff is on TCP or UDP; although I would be surprised if the CVO does anything on UDP.

[1]: https://github.com/openshift/installer/blob/499f53fcbc89fdddd5541f00a9c97578bb7dbc6d/data/data/gcp/network/firewall.tf#L112-L116

**Review by wking — COMMENTED** [review]
Thanks for this :).  Left a few nits inline.

**Comment by kalexand-rh** [comment]
Thank you for the review! I think I've fixed all your requests. Please let me know if you see anything else.

**Comment by kalexand-rh** [comment]
@jianlinliu, will you please take a look at these changes?

File: modules/installation-network-user-infra.adoc [hunk 9]
```diff
@@ -48,6 +53,18 @@ the Cluster Version Operator on port `9099`.
...
+.4+|UDP
+|`4789`
+|VXLAN and GENEVE
+
+|`6081`
+|VXLAN and GENEVE
+
+|`9000`-`9999`
+|Host level services, including the node exporter on ports `9100`-`9101` and
+the Cluster Version Operator on port `9099`.
```
Comment (wking) [inline modules/installation-network-user-infra.adoc]: Let me check, but I think we should drop the CVO here.

Comment (wking) [reply modules/installation-network-user-infra.adoc]: Confirmed 9099 since openshift/cluster-version-operator@0bb98c9e67b48b672fb947ea5728f66699deadfc , but it's just TCP for metrics, not this UTC entry.

**Comment by jianlinliu** [comment]
lgtm.

File: modules/installation-aws-user-infra-rhcos-ami.adoc [hunk 10]
```diff
@@ -18,48 +18,48 @@ You must use a valid {op-system-first} AMI for your Amazon Web Services
 |AWS AMI
```
Comment (jianlinliu) [inline modules/installation-aws-user-infra-rhcos-ami.adoc]: This is for 4.2, right? If yes, the boot image ami should be aligned with https://github.com/openshift/installer/blob/release-4.2/data/data/rhcos.json.

Comment (kalexand-rh) [reply modules/installation-aws-user-infra-rhcos-ami.adoc]: Thank you @jianlinliu! Please let me know if you see other changes.

**Comment by jianlinliu** [comment]
LGTM.

File: modules/installation-network-user-infra.adoc (hunk 9 above)
Comment (kalexand-rh) [reply modules/installation-network-user-infra.adoc]: I'm dropping the CVO reference here. Thanks!

**Comment by kalexand-rh** [comment]
@openshift/team-documentation PTAL

File: installing/installing_aws/installing-aws-customizations.adoc [hunk 11]
```diff
@@ -1,14 +1,14 @@
 [id="installing-aws-customizations"]
 = Installing a cluster on AWS with customizations
 include::modules/common-attributes.adoc[]
-:context: install-customizations-cloud
+:context: installing-aws-customizations
```
Comment (bergerhoffer) [inline installing/installing_aws/installing-aws-customizations.adoc]: I see two modules where install-customizations-cloud is still used in ifeval statements

File: modules/architecture-platform-benefits.adoc [hunk 12]
```diff
@@ -40,19 +32,16 @@ unique features and benefits of {product-title}.
...
 
 {op-system} includes:
 
-* Ignition, which is a firstboot system configuration for initially bringing up and
-configuring {product-title} nodes.
-* cri-o, a Kubernetes native container runtime implementation that integrates
-closely with the operating system to deliver an efficient and optimized
-Kubernetes experience.
+* Ignition, which {product-title} uses as a firstboot system configuration for initially bringing up and configuring machines.
+* CRI-O, a Kubernetes native container runtime implementation that integrates closely with the operating system to deliver an efficient and optimized Kubernetes experience. CRI-O provides facilities for running, stopping, and restarting containers. It fully replaces the Docker Container Engine in {product-title} {product-version}.
```
Comment (bergerhoffer) [inline modules/architecture-platform-benefits.adoc]: This last sentence seems unclear. It's hard to tell whether you're saying that Docker Container Engine is in 4.2, or that in 4.2 it was replaced. Suggest rewording for clarity, maybe saying that it replaces Docker Container Engine, which was used in previous (maybe specifically say 3?) versions of OCP?

File: modules/installation-aws-user-infra-requirements.adoc [hunk 13]
```diff
@@ -37,9 +39,9 @@ control plane initializes and you can access the cluster API by using the `oc`
 command line interface.
 ////
 
-You can use the following instance types for the cluster machines:
+You can use the following instance types for the cluster machines with the provided CloudFormation templates.
 
-.Valid instance types for machines
+.Instance types for machines
```
Comment (bergerhoffer) [inline modules/installation-aws-user-infra-requirements.adoc]: The way that there is an important block in between this heading and the table makes it render that this title is for the note, not the table. The heading needs to be with the table - suggest moving the important admonition above the heading.

File: modules/installation-initializing.adoc [hunk 14]
```diff
@@ -131,9 +142,7 @@ endif::gcp[]
 link:https://cloud.redhat.com/openshift/install[OpenShift Infrastructure Providers] page.
 
 . Modify the `install-config.yaml` file. You can find more information about
-the available parameters in the *Installation configuration parameters* section
-and in the
-link:https://godoc.org/github.com/openshift/installer/pkg/types#InstallConfig[Go documentation].
+the available parameters in the *Installation configuration parameters* section.
```
Comment (bergerhoffer) [inline modules/installation-initializing.adoc]: I don't think bold is appropriate here, that's used pretty much for GUI items. Suggest italics instead.

File: modules/ssh-agent-using.adoc [hunk 15]
```diff
@@ -72,5 +89,6 @@ Identity added: /home/<you>/<path>/<file_name> (<computer_name>)
 .Next steps
 
 When you install {product-title}, provide the SSH public key to the installation program.
```
Comment (bergerhoffer) [inline modules/ssh-agent-using.adoc]: I assume "Next steps" should use the same format as "Additional resources", and this should be a bullet.

**Review by bergerhoffer — COMMENTED** [review]
Just a few minor things!

File: modules/installation-initializing.adoc (hunk 14 above)
Comment (kalexand-rh) [reply modules/installation-initializing.adoc]: Italics is supposed to be for emphasis on first use of terms, and bold can also be used sparingly for generic emphasis. https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#quick-markup-reference
